### PR TITLE
[PUBDEV-4739] - h2o.sub now retains column names

### PR DESCRIPTION
**Pull request by michal-raska**
There was a missing arg when creating new Frame. I have also added a simple test.

### CODE REVIEW

**Comment by michal-raska** [comment]
@h2o-ops please test!

**Comment by michal-raska** [comment]
The test is not working, will fix it tomorrow, sorry for premature review requests

File: h2o-r/tests/testdir_jira/runit_pubdev_4739.R [hunk 1]
```diff
@@ -0,0 +1,17 @@
+setwd(normalizePath(dirname(R.utils::commandArgs(asValues=TRUE)$"f")))
+source("../../scripts/h2o-r-test-setup.R")
+
+
+
+test.pubdev_4739 <- function() {
+    voting.url <- "http://archive.ics.uci.edu/ml/machine-learning-databases/voting-records/house-votes-84.data"
```
Comment (navdeep-G) [inline h2o-r/tests/testdir_jira/runit_pubdev_4739.R]: Please try to use `/smalldata` directory for fetching datasets. URL's are not the most stable solution.

**Comment by michal-raska** [comment]
@h2o-ops please test!

**Comment by michal-raska** [comment]
@navdeep-G Dataset is being fetched from smalldata folder.
The test looks OK now.

**Comment by michal-raska** [comment]
@h2o-ops please test!

**Comment by michal-raska** [comment]
@h2o-ops please test!

**Pull request merged**